staffs/administrator.cs

```diff
@@ -1,11 +1,58 @@
 
 using System;
 namespace training{
-    class Administrator :Staff{
-        public override void add(){
+    public class Administrator :Staff{
+        private string administratorDepartment;
+
+        public string AdministratorDepartment{
+            get{return administratorDepartment;}
+            set{administratorDepartment=value;}
```

Comment:
Since there is no additional logic implemented here, empty setters are fine here

Reply:
ex: public string AdministratorDepartment{get;set;}, Compiler automatically does what you did here

Reply:
didnt get the meaning of empty setters. can you give me an example code.